src/views/pacbio/PacbioPoolIndex.vue

```diff
@@ -60,7 +60,7 @@
 
         <template #row-details="row">
           <div>
-            <traction-table :items="row.item.libraries" :fields="field_in_details">
+            <traction-table :items="row.item.used_aliquots" :fields="field_in_details">
```

Comment:
Tested this and I don't think this is working properly.
This shows we are missing some tests for this behaviour as well. Maybe worth adding some to the cypress tests for this page.

Reply:
Good finding. I will have a look.

Reply:
Fixed